flight/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""Initialization file for state exporting"""
+from typing import Dict
+from states.state import State
+from states.start_state import Start
+from states.pre_processing import PreProcess
+from states.takeoff import Takeoff
+from states.waypoints import Waypoints
+from states.land import Land
+from states.final_state import Final
```

Comment:
These imports should be absolute imports relative to the root directory of the repo.
Ex. `from flight.states.state import State`